7-bank-project/3-data/translations/assignment.fr.md

```diff
@@ -0,0 +1,15 @@
+# Refactorisez et commentez votre code
+
+## Instructions
+
+Au fur et à mesure que votre codebase se développe, il est important de remanier votre code fréquemment pour qu'il reste lisible et maintenable dans le temps. Ajoutez des commentaires et remaniez votre `app.js` pour améliorer la qualité du code :
```

Comment:
What do you think about : "Au fur et à mesure que votre base de code se développe" instead of "Au fur et à mesure que votre codebase se développe" ?

And "refactorisez" isn't better than "remaniez" in this context?

Reply:
I think you are right, I updated the document. Thank you